### PR TITLE
🪟 🧪E2E testing: DB data manipulation + Catalog Diff modal test cases

### LINKED ISSUE #19430: E2E testing: investigate the possibility to update the Postgres DB in order to cover test cases with Diff modal
In order to cover test cases with Diff modal(when user is notified about changes in Source) we need to update the data during the Cypres test run.

### PR DESCRIPTION
**Pull request by dizel852**
## What
Resolves [19430](https://github.com/airbytehq/airbyte/issues/19430)

## How
Adds integration with covering Catalog Diff modal cases:
- new stream is added
- stream is removed
- a stream has changed: some columns were removed and some were added

Cypress test run: https://dashboard.cypress.io/projects/916nvw/runs/14637/overview

The whole test case:
1. Create a connection `Postgres => Local JSON`
2. Change data in DB
3. Open connections settings (Replication Tab)
4. Click the "Refresh Source Schema" button
5. 💡 Catalog Diff modal should appear
6. Select newly added stream
7. Click the "Save changes" button
8. 💡 "Stream configuration changed" modal should appear
9. Confirm reset
10. Remove the created connection
11. Clean DB from populated data

+ add utility functions for manipulation(create, update delete) data in Postgres DB directly from test run

Screenshot at Nov 16 02-23-20

### CODE REVIEW

File: airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts [hunk 1]
```diff
@@ -201,6 +208,54 @@ describe("Connection main actions", () => {
...
+    goToReplicationTab();
+    refreshSourceSchemaBtnClick();
+
+    cy.get("[data-testid='catalog-diff-modal']").should("exist");
+
+    cy.get("table[aria-label='removed streams table']").should("contain", "users");
+
+    cy.get("table[aria-label='new streams table']").should("contain", "cars");
+
+    cy.get("button[aria-label='toggle accordion']").click();
```
Comment (SofiiaZaitseva) [inline airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts:237]: can we move all elements to the pages level and create methods for working with these elements? so everyone can reuse it and we won't duplicate code

File: airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts [hunk 2]
```diff
@@ -201,6 +208,54 @@ describe("Connection main actions", () => {
...
+    cy.get("table[aria-label='removed fields']").should("contain", "city_code");
+    cy.get("table[aria-label='new fields']").children().should("contain", "country").and("contain", "state");
+
+    updateSchemaModalConfirmBtnClick();
+
+    cy.get("[data-testid='cars-stream-sync-checkbox']").check({ force: true });
```
Comment (SofiiaZaitseva) [inline airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts:243]: the same

**Review by SofiiaZaitseva — CHANGES_REQUESTED** [review]

File: airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts (hunk 1 above)
Comment (dizel852) [reply airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts:237]: Duly noted! Will fix that! thanks)

File: airbyte-webapp-e2e-tests/cypress/plugins/index.ts [hunk 3]
```diff
@@ -12,10 +12,16 @@
...
 // the project's config changing)
 
+import Cypress from "cypress"
+
+
 /**
  * @type {Cypress.PluginConfig}
  */
-module.exports = (on, config) => {
+module.exports = (on: Cypress.PluginEvents, config: Cypress.PluginConfigOptions) => {
```
Comment (krishnaglick) [inline airbyte-webapp-e2e-tests/cypress/plugins/index.ts:21]: 👍

**Review by krishnaglick — APPROVED** [review]
Code LGTM. I did not test locally but given this is an addition to CI I imagine CI will let us know if there's an issue ;)

**Review by SofiiaZaitseva — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**